Assets/Scripts/StandardSamples/SignInWithGoogle/SignInWithGoogleManager.cs

```diff
@@ -94,6 +95,12 @@ public void GetGoogleIdToken(System.Action<string, string> callback)
             javaCallback.callback = callback;
 
             AndroidConfig config = AndroidConfig.Get<AndroidConfig>();
+
+            if (config.GoogleLoginClientID == null) 
```

Comment:
Can this check String.IsNullOrEmpty instead of checking directly to null?

Reply:
Done!